packages/free/src/adapter.ts

```diff
@@ -108,7 +108,12 @@ async function retryFetch(
     res = await fetch(...args);
     if (res.status >= 500) {
       console.error(`${res.status} in free session service, retrying!`);
+      await sleep(3000);
```

Comment:
Is such a big sleep needed there?

Reply:
What else do you suggest?

Reply:
> What else do you suggest?

Increase the time as the number of requests increases. Or do you not like this approach?

Reply:
Yes, we can do that. Linearly? Quadratically? Exponentially?

Reply:
> Yes, we can do that. Linearly? Quadratically? Exponentially?

Thats up to you.

Reply:
https://t.me/grammyjs/99895

Reply:
See #136